src/imitation/policies/replay_buffer_wrapper.py

```diff
@@ -0,0 +1,83 @@
+"""Wrapper for reward labeling for transitions sampled from a replay buffer."""
+
+
+from typing import Mapping, Optional, Type
+
+import numpy as np
+import torch as th
+from stable_baselines3.common.buffers import ReplayBuffer
+from stable_baselines3.common.type_aliases import ReplayBufferSamples
+
+from imitation.rewards.reward_function import RewardFn
+from imitation.util import util
+
+
+def _samples_to_reward_fn_input(
+    samples: ReplayBufferSamples,
+) -> Mapping[str, np.ndarray]:
+    """Convert a sample from a replay buffer to a numpy array."""
+    return dict(
+        state=samples.observations.cpu().numpy(),
+        action=samples.actions.cpu().numpy(),
+        next_state=samples.next_observations.cpu().numpy(),
+        done=samples.dones.cpu().numpy(),
+    )
+
+
+class ReplayBufferRewardWrapper(ReplayBuffer):
+    """Relabel the rewards in transitions sampled from a ReplayBuffer."""
+
+    def __init__(
+        self,
+        *args,
+        replay_buffer_cls: Type[ReplayBuffer] = ReplayBuffer,
+        reward_fn: Optional[RewardFn] = None,
+        **kwargs,
+    ):
+        """Builds ReplayBufferRewardWrapper.
+
+        Note(yawen): we directly inherit ReplayBuffer in this case and leave out the
```

Comment:
Including your username is better than just leaving an anonymous note (gives us someone to follow up with), but I think better yet is to link to the GitHub issue/PR so people have surrounding context. You could link to the permalink https://github.com/HumanCompatibleAI/imitation/pull/459#issuecomment-1201997194?

I also think this should maybe be a comment not a docstring, as it's more an implementation detail not part of public API docs, but close call -- don't feel strongly about this.

Reply:
+1